src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -70,6 +70,20 @@
             field.classList.toggle('ibexa-available-field-type--hidden', isFieldHidden);
         });
     };
+    const reinitIdentifierAutogenerator = (fieldNode) => {
+        const sourceInputs = fieldNode.querySelectorAll('[data-autogenerate-identifier-target-selector]');
+        sourceInputs.forEach((sourceInput) => {
+            const { autogenerateIdentifierTargetSelector } = sourceInput.dataset;
+            const targetInput = doc.querySelector(autogenerateIdentifierTargetSelector);
+            const identifierAutogenerator = new ibexa.core.SlugValueInputAutogenerator({
+                sourceInput,
+                targetInput,
+                whitespaceTextReplacer: '_',
```

Comment:
There is no need to pass `whitespaceTextReplacer: '_'`, because by default it is set to `_`
https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/public/js/scripts/core/slug.value.input.autogenerator.js#L6